ttnn/tracy/__init__.py

```diff
@@ -183,164 +183,3 @@ def get_available_port():
 
 def split_comma_list(option, opt, value, parser):
     setattr(parser.values, option.dest, value.split(","))
-
```

Comment:
not needed anymore?

Reply:
This was all defined in `__main__.py` it was all repeated here.